src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerMatchResource.kt

```diff
@@ -23,16 +23,16 @@ import javax.validation.Valid
 @RestController
 @Validated
 @RequestMapping(value = ["/match-prisoners"], produces = [MediaType.APPLICATION_JSON_VALUE])
-@PreAuthorize("hasRole('ROLE_GLOBAL_SEARCH')")
+@PreAuthorize("hasAnyRole('GLOBAL_SEARCH','ROLE_PRISONER_SEARCH')")
```

Comment:
missing `ROLE_` prefix?

Reply:
looks like the tests should that it works for both, but for consistency worth just using `ROLE_GLOBAL_SEARCH` everywhere instead

Reply:
updated